package.json

```diff
@@ -1,116 +1,120 @@
-    "clear-iv": "NODE_ENV=localhost node ./server/scripts/clear-in-verification-field.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/HelpAssistHer/help-assist-her.git"
-  },
-  "author": "Karen M. Rose",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/HelpAssistHer/help-assist-her/issues"
-  },
-  "homepage": "https://github.com/HelpAssistHer/help-assist-her#readme",
-  "dependencies": {
-    "@google/maps": "^0.3.1",
-    "@turf/turf": "^3.7.4",
-    "axios": "0.15.2",
-    "babel-cli": "^6.24.1",
-    "babel-core": "6.16.0",
-    "babel-loader": "6.2.7",
-    "babel-plugin-syntax-async-functions": "^6.13.0",
-    "babel-plugin-transform-async-to-generator": "^6.22.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.18.0",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-plugin-transform-regenerator": "^6.24.1",
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-react": "6.16.0",
-    "babel-register": "6.16.0",
-    "bluebird": "^3.4.7",
-    "body-parser": "^1.16.1",
-    "boom": "^4.3.1",
-    "chai": "^3.5.0",
-    "chai-http": "^3.0.0",
-    "cheerio": "^1.0.0-rc.1",
-    "colors": "^1.1.2",
-    "compression": "^1.6.2",
-    "config": "^1.24.0",
-    "connect-mongo": "^1.3.2",
-    "cookie-parser": "^1.4.3",
-    "cors": "^2.8.1",
-    "csv": "^2.0.0",
-    "express": "^4.15.3",
-    "express-boom": "^2.0.0",
-    "express-session": "^1.15.2",
-    "history": "^4.4.0",
-    "joi": "^10.4.1",
-    "joi-phone-validator": "^1.0.0",
-    "json-loader": "0.5.4",
-    "lodash": "^4.17.4",
-    "log": "^1.4.0",
-    "mapbox": "^1.0.0-beta5",
-    "mocha": "^3.2.0",
-    "moment": "^2.18.1",
-    "mongodb": "^2.2.16",
-    "mongodb-extended-json": "^1.9.0",
-    "mongoose": "^5.0.16",
-    "morgan": "^1.8.1",
-    "passport": "^0.3.2",
-    "passport-facebook-token": "^3.3.0",
-    "preact": "^6.4.0",
-    "preact-compat": "^3.9.3",
-    "ramda": "^0.25.0",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
-    "react-facebook-login": "^3.6.1",
-    "react-jss": "^6.1.1",
-    "react-redux": "^5.0.4",
-    "react-router-dom": "^4.1.1",
-    "redux": "^3.6.0",
-    "redux-form": "^7.1.2",
-    "redux-thunk": "^2.2.0",
-    "webpack": "^2.5.1"
-  },
-  "devDependencies": {
-    "babel-jest": "16.0.0",
-    "babel-preset-env": "^1.6.1",
-    "css-loader": "0.25.0",
-    "eslint": "^3.19.0",
-    "eslint-config-standard": "6.1.0",
-    "eslint-config-standard-jsx": "3.1.0",
-    "eslint-config-standard-react": "4.1.0",
-    "eslint-loader": "^1.7.1",
-    "eslint-plugin-promise": "2.0.1",
-    "eslint-plugin-react": "6.3.0",
-    "eslint-plugin-standard": "2.0.0",
-    "jest": "15.1.1",
-    "jsdom": "9.5.0",
-    "react-addons-test-utils": "15.3.2",
-    "react-test-renderer": "15.3.2",
-    "style-loader": "0.13.1",
-    "webpack-dev-server": "^2.4.5"
-  }
+	"name": "help-assist-her",
```

Comment:
Prettier ran on this file and added tabs 😄 
The changes I made are:
- Alphabetize the scripts
- Add the precommit script
- Install the prettier, pretty-quick, and husky packages (all needed for Prettier)